test/integration_test/cloud_provider/pure.go

```diff
@@ -0,0 +1,24 @@
+package cloud_provider
+
+type pureProvider struct{}
+
+func newPureProvider() Provider {
+	return pureProvider{}
+}
+
+func (pureProvider) GetDefaultDataDrives() *[]string {
+	return stringSlicePtr([]string{
+		"type=pure-block,size=150"})
+}
+
+func (pureProvider) GetDefaultMetadataDrive() *string {
+	return stringPtr("type=pure-block,size=64")
+}
+
+func (pureProvider) GetDefaultKvdbDrive() *string {
+	return stringPtr("type=pure-block,size=32")
+}
+
+func (pureProvider) GetDefaultJournalDrive() *string {
+	return stringPtr("type=pure-block,size=32")
+}
```

Comment:
From where did you get these specs? Asking to help with review.

Reply:
The type=pure-block spec I got from the output below:-
`[root@ragsingh-13-2 /]# pxctl cd inspect --node 5bd16486-e001-4bfe-ac30-0b5f7884fc03

Drive Set Configuration
        Number of drives in the Drive Set:  3
        NodeID:  5bd16486-e001-4bfe-ac30-0b5f7884fc03
        NodeIndex:  1
        InstanceID:  ragsingh-13-2
        Zone:  default
        State:  In Use
        Labels:  -

        Drive  0
                ID:  01173e50e8
                Type:  pure-block
                Size:  150 Gi
                Iops:  0
                Path:  /dev/mapper/3624a93705ef29a2003c54fac0008afb3
                Used for:  data
                Labels:  portworx.io/flasharray-id=5ef29a20-03c5-4fac-acc2-4c8d348ed87d,portworx.io/flasharray-management-endpoint=dogfood-roast.dev.purestorage.com,portworx.io/flasharray-name=dogfood-roast

        Drive  1
                ID:  db3935a0e2
                Type:  pure-block
                Size:  150 Gi
                Iops:  0
                Path:  /dev/mapper/3624a93705ef29a2003c54fac0008afb8
                Used for:  data
                Labels:  portworx.io/flasharray-id=5ef29a20-03c5-4fac-acc2-4c8d348ed87d,portworx.io/flasharray-management-endpoint=dogfood-roast.dev.purestorage.com,portworx.io/flasharray-name=dogfood-roast

        Drive  2
                ID:  ed6570d647
                Type:  pure-block
                Size:  32 Gi
                Iops:  0
                Path:  /dev/mapper/3624a93705ef29a2003c54fac0008afb5
                Used for:  kvdb
                Labels:  portworx.io/flasharray-id=5ef29a20-03c5-4fac-acc2-4c8d348ed87d,portworx.io/flasharray-management-endpoint=dogfood-roast.dev.purestorage.com,portworx.io/flasharray-name=dogfood-roast
[root@ragsingh-13-2 /]# `

The size for DefaultDataDrives and DefaultKvdbDrive is mentioned in the storagecluster spec that we install.
The size for DefaultJournalDrive is mentioned in https://docs.portworx.com/portworx-enterprise/operations/operate-other/performance-and-tuning/tuning.html. I will make the change from 32 to 3GB.
The size for DefaultMetadataDrive is mentioned at https://docs.portworx.com/portworx-enterprise/install-portworx/kubernetes/bare-metal/install-px-store-v2.html and https://purestorage.slack.com/archives/C01NDR7ERPA/p1643810887323779?thread_ts=1643808850.224219&cid=C01NDR7ERPA

Please let me know if I have the specs correct.

Reply:
@harsh-px Can you let me know if the specs I picked are correct?